src/about/hackathon.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+
+title: Hackathon
+description: >
+  We're thrilled to announce the first edition of the DjangoCon Hackathon, happening this year.
+  This is your chance to learn new skills, collaborate with friendly Django developers,
+  and showcase your creativity.
+
+permalink: /hackathon/
+---
+
+<div class="block-container">
+  <div class="wrapper">
+    <header class="flex flex-wrap items-center">
+      <div class="flex-1 max-w-screen-lg">
+        <h1 class="mb-12 pageheading">{{ title }}</h1>
+        <p class="subheading">
+          Welcome to DjangoCon US 2024 Hackathon!
+        </p>
+      </div>
+    </header>
+  </div>
+</div>
+
+<div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="grid-1-2">
+      <div class="prose lg:prose-lg">
+{% markdown %}
+Are you a beginner eager to dive into Django development? Or a seasoned pro ready to take on new challenges? No matter your skill level, this event is for you!
+
+We're thrilled to announce the first edition of the DjangoCon Hackathon, happening this year. This is your chance to learn new skills, collaborate with friendly Django developers, and showcase your creativity.
+
+Join us for a day of coding, learning, and fun. Let's build something extraordinary together!
+{% endmarkdown %}
+
+      <a class="button text-gray-500 cursor-not-allowed pointer-events-none" href="#">Registration starts on August 20!</a>
+      </div>
+
+      <section class="relative flex flex-col self-start p-6 prose bg-white lg:prose-lg bg-lines lg:pt-10 card-effect-3">
+{% markdown %}
+## What to expect:
+
+-   **Registration:** Registration will be available starting August 20, 2024
+-   **Announcement of hackathon participants:** All accepted hackathon participants will receive their acceptance emails by 3rd September, 2024
+-   **Slack group:** All accepted hackathon participants will be added to the hackathon slack group for easier communication
+-   **Hackathon launch:** The hackathon will officially begin on 19th September, 2024 and will go on till 26th September, 2024
+-   **Individual or Group Projects:** Work solo or team up to create something amazing
+-   **Expert Mentors:** Our fantastic mentors will be there to guide you every step of the way
+-   **Networking Opportunities:** Connect with fellow developers and expand your professional network
+-   **Presentation:** Each team or individual will have the opportunity to showcase their project to both the participants and judges and get feedback. This will take place on the last day of the hackathon ie, 26th September, 2024
```

Comment:
```suggestion
-   **Presentation:** Each team or individual will have the opportunity to showcase their project to both the participants and judges and get feedback. This will take place on the last day of the hackathon, September 26th, 2024
```